src/style.css

```diff
@@ -0,0 +1,27 @@
+body {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background: url(./images/star.svg) no-repeat center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0),
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background: no-repeat center url("../src/images/star-active.svg");
```

Comment:
You can modify here just **background-image** as stars_star already have **bacground-repeat** and **backround-position** property